lib/ecto/adapters/postgres.ex

```diff
@@ -18,7 +18,7 @@ defmodule Ecto.Adapters.Postgres do
   Postgres options split in different categories described
   below. All options can be given via the repository
   configuration:
-  
+
       config :your_app, YourApp.Repo,
         ...
 
```

Comment:
I would document this new option in the sections below :)

Reply:
That would be great :)